freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/UserSyncService.java

```diff
@@ -208,8 +208,10 @@ private void internalSynchronizeUsers(String operationId, String accountId, Stri
                     umsEventGenerationIdsProvider.getEventGenerationIds(accountId, requestId) :
                     null;
 
+            LOGGER.debug("Starting RETRIEVE_{}_UMS_STATE for environments {} ...", fullSync ? "FULL" : "PARTIAL", environmentCrns);
```

Comment:
Is there a request ID in this context? If there is, is it included in the message? Maybe add the operation ID?

Reply:
the request id and operation id (exposed as "flow") are part of the MDC.
```
2020-07-13 14:42:58,291 [usersyncExecutor-1] lambda$internalSynchronizeUsers$7:204 DEBUG c.s.f.s.f.u.UserSyncService - [instance:default] [type:springLog] [crn:] [name:] [flow:ab77043f-84e5-49d1-9e57-fa27e8393a9f] [requestid:2bc3eed0-77ff-4ab2-b5a2-2ce4fd17da4c] [tenant:] [userCrn:] [environment:] [traceId:] [spanId:] Starting RETRIEVE_FULL_UMS_STATE for environments [crn:cdp:environments:us-west-1:foo:environment:8813ff3f-b25b-40f8-9999-569804d48396] ...
```